src/pages/BorrowerProfile/FundedBorrowerProfile.vue

```diff
@@ -86,9 +86,99 @@
 				</div>
 			</div>
 		</div>
-		<div v-else id="loading-overlay">
-			<div class="spinner-wrapper">
-				<kv-loading-spinner />
+		<div v-else>
+			<article class="tw-relative tw-bg-secondary">
+				<div class="tw-relative">
+					<div class="tw-absolute tw-top-0 tw-h-full tw-w-full tw-overflow-hidden">
+						<hero-background />
+					</div>
+					<kv-page-container>
+						<kv-grid class="md:tw-grid-cols-12 tw-grid tw-relative">
+							<div class="md:tw-col-span-8 md:tw-col-start-3 tw-my-8">
+								<section class="
+									tw-pb-0
+									md:tw-bg-primary
+									md:tw-pb-2.5
+									tw-py-2.5 md:tw-p-3 lg:tw-p-4
+									md:tw-rounded-t lg:tw-rounded"
```

Comment:
This should be only `md:tw-rounded` instead of `md:tw-rounded-t lg:tw-rounded` so that the corners are always rounded.